matchers/init_test.go

```diff
@@ -3,15 +3,30 @@ package matchers_test
 import (
 	"testing"
 
+	"github.com/paketo-buildpacks/occam"
 	"github.com/sclevine/spec"
 	"github.com/sclevine/spec/report"
+
+	. "github.com/onsi/gomega"
 )
 
 func TestUnitMatchers(t *testing.T) {
+	var Expect = NewWithT(t).Expect
+
+	docker := occam.NewDocker()
+	err := docker.Pull.Execute("alpine:latest")
```

Comment:
I don't love that the test setup has knowledge of the implementation of the specific tests (i.e. that they use `alpine:latest`). But, this is how it was done in `jam` and I can't come up with a better architecture at this time. So I'm happy with it 👍